### PR TITLE
added documentation on padrino environment and reload configuration

### PR DESCRIPTION
**Pull request by u007**
https://github.com/padrino/padrino-framework/issues/2049#issuecomment-229626178

### CODE REVIEW

File: guides/02_getting-started/04_blog-tutorial.html.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ For now, the defaults for the database connection settings (`config/database.rb`
 OK for this tutorial. A sqlite3 database will be used that is stored inside
 `db/sample_blog_development.db`.
 
-To setup environment, set RACK_ENV=enviromentname. This environment can be configured in config/apps.rb as:
```
Comment (wikimatze) [inline guides/02_getting-started/04_blog-tutorial.html.md]: I like the spelling in this commit with the sentence `This environment can be configured in config/apps.rb as:` better than the new one.

File: guides/02_getting-started/04_blog-tutorial.html.md [hunk 2]
```diff
@@ -144,7 +144,7 @@ Padrino.configure_apps do
   end
 end
 ```
-or can be set in app/app.rb as
```
Comment (wikimatze) [inline guides/02_getting-started/04_blog-tutorial.html.md]: Same as above, grammar sounds better in this case.

**Comment by nesquena** [comment]
@wikimatze should we merge this?

File: guides/02_getting-started/04_blog-tutorial.html.md (hunk 1 above)
Comment (u007) [inline guides/02_getting-started/04_blog-tutorial.html.md]: done

**Comment by wikimatze** [comment]
That's fine and it's already merged :).

**Pull request merged**